main.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/dxw/git-test-branch/screenwriter"
```

Comment:
Can't this be a relative path instead of a pull path?

Reply:
Nope. That's intentionally not an option in Go.

Reply:
It's been a while :). I think it does work, but breaks `go get` for the package, right?